pxtpy/pydecompiler.ts

```diff
@@ -1034,6 +1034,12 @@ namespace pxt.py {
                 return exps.map((el, i) => {
                     let sep = el.charAt(el.length - 1) == "," ? "" : ",";
                     if (i == 0) {
+                        let lines = el.split('\n');
+                        // for multiline element, indent all lines after first
+                        if (lines.length > 1) {
+                            let first = lines.shift();
+                            el = first + '\n' + indent1(lines.join('\n'));
+                        }
```

Comment:
Do you still need this since you made the change to "indent" above?

Reply:
this skips the first line for cases like 

```
let arr = [img("""
    . . .  """)]
```

where we don't want to indent the `img("""` part

Reply:
Ohh gotcha